iommi/_db_compat.py

```diff
@@ -48,7 +48,19 @@ def setup_db_compat_django():
     )
 
     # The order here is significant because of inheritance structure. More specific must be below less specific.
```

Comment:
This comment should be next to the register_factory calls

Reply:
Fixed